### PR TITLE
Add onboarding config

### PR DESCRIPTION
**Pull request by nickcernis**
This PR enables automatic redirect to the new homepage setup admin page in Genesis 2.8.0+ upon theme activation.

When the theme is activated people will then be prompted to automatically install Atomic Blocks and a block-based static homepage.

To-do:
- [x] Update homepage layout
- [x] Fix issue with the Atomic Blocks CTA block reporting “this block contains unexpected or invalid content” upon import. (Same as https://wordpress.org/support/topic/bug-cta-block-title-text-font-size-unexpected-output/.) Also noting that block will be refactored so may need to be updated in `homepage.php`. https://github.com/ArrayHQ/atomic-blocks/issues/21

For #150.

### CODE REVIEW

**Comment by nickcernis** [comment]
@mindctrl Do you have time to review this, please? Since you last looked I have:

- Altered the homepage layout with new blocks and images.
- Adjusted styling.

There are still some improvements that can be made to the homepage styling, but I'm happy with where this is at for a beta release.

File: config/import/content/homepage.php [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+<p>Describe yourself or your business in a short sentence or two.</p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:column -->
+
+<!-- wp:column -->
+<div class="wp-block-column"><!-- wp:heading {"level":4} -->
+<h4>Share this page</h4>
+<!-- /wp:heading -->
+
+<!-- wp:atomic-blocks/ab-sharing {"google":false,"linkedin":true,"email":true,"shareButtonStyle":"ab-share-icon-only"} /--></div>
```
Comment (mindctrl) [inline config/import/content/homepage.php]: @nickcernis the AB Sharing block notice can be fixed by adding `"shareAlignment":"center"` to this line. I think this is a bug in AB, but it'll fix it here for us now.

Comment (mindctrl) [reply config/import/content/homepage.php]: Edited by above comment to say `"shareAlignment":"left"`, I accidentally pasted `center` from a test I was doing.

**Comment by mindctrl** [comment]
Aside from the note about the AB Sharing block, this looks good and is working as expected. Once that's fixed up, we can merge this.

File: config/import/content/homepage.php (hunk 1 above)
Comment (nickcernis) [reply config/import/content/homepage.php]: Done — nice catch and thanks, @mindctrl!

**Pull request merged**